cmd/dagger/main.go

```diff
@@ -399,14 +410,14 @@ const usageTemplate = `{{ "Usage" | toUpperBold }}
 
 {{- if .HasAvailableLocalFlags}}
 
-{{ "Flags" | toUpperBold }}
+{{ "Options" | toUpperBold }}
 {{ flagUsagesWrapped .LocalFlags | trimTrailingWhitespaces}}
 
 {{- end}}
 
 {{- if .HasAvailableInheritedFlags}}
 
-{{ "Global Flags" | toUpperBold }}
+{{ "Global Options" | toUpperBold }}
```

Comment:
This is more accurate and is what `gh` uses:
```suggestion
{{ "Inherited Options" | toUpperBold }}
```